src/ApiLoader/EnableInTracee.cpp

```diff
@@ -122,6 +123,9 @@ ErrorMessageOr<void> SetApiEnabledInTracee(const CaptureOptions& capture_options
     // Call "orbit_api_set_enabled" in tracee.
     OUTCOME_TRY(ExecuteInProcess(pid, orbit_api_set_enabled_function, function_table_address,
                                  api_function.api_version(), enabled ? 1 : 0));
+    // `orbit_api_set_enabled` could spawn new threads; stop those too.
+    OUTCOME_TRY(already_attached_tids,
+                AttachAndStopNewThreadsOfProcess(pid, already_attached_tids));
```

Comment:
Why is it important to stop the threads spawned internally by `orbit_api_set_enabled`? Aren't we going to resume them right after? Is that for the case where those threads contained manual instrumentation? Maybe clarify why this is done in the comment.

Reply:
I expanded the comment. The threads are not going to resume right after if the loop is executed again. The loop calls `ExecuteInProcess` which assumes a completely stopped process. We had a very similar case with user space instrumentation and if the process is not completely stopped this can lead to crashes.